TombEngine/Scripting/Internal/TEN/Misc/Miscellanous.cpp

```diff
@@ -262,6 +263,18 @@ namespace Misc
 		return std::make_tuple(resX, resY);
 	}
 
+	/// Reset object camera back to Lara and deactivate object camera.
+	//  ObjCamera @tparam camSlotId 0 for Lara
```

Comment:
Comments are formatted in a wrong way. This will corrupt autogenerated documentation. 
Please look at other functions with similar comments to get an idea how to compose such comments.

Reply:
changed formatting